--- gnustep-back-0.9.5/Fonts/GNUmakefile.postamble.orig	2005-02-23 04:47:34.000000000 +0100
+++ gnustep-back-0.9.5/Fonts/GNUmakefile.postamble	2005-04-24 23:08:07.545380568 +0200
@@ -35,7 +35,7 @@
 # Things to do after installing
 after-install::
 	for file in $(FONT_INSTALL_FILES); do \
-	  ($(TAR) cfX - $(GNUSTEP_MAKEFILES)/tar-exclude-list $$file) | (cd $(GNUSTEP_SYSTEM_ROOT)/$(back-resources_RESOURCE_FILES_INSTALL_DIR); $(TAR) xf -) ;\
+	  ($(TAR) cfX - $(GNUSTEP_MAKEFILES)/tar-exclude-list $$file) | (cd $(GNUSTEP_INSTALLATION_DIR)/$(back-resources_RESOURCE_FILES_INSTALL_DIR); $(TAR) xf -) ;\
 	done
 
 # Things to do before uninstalling
