--- gnustep-back-0.8.3/Headers/xlib/XftFontInfo.h.orig	Thu Mar 28 00:45:33 2002
+++ gnustep-back-0.8.3/Headers/xlib/XftFontInfo.h	Sun Dec 29 12:28:04 2002
@@ -34,7 +34,7 @@
 
 #include <AppKit/GSFontInfo.h>
 
-@interface XftFontInfo : GSFontInfo
+@interface GSXftFontInfo : GSFontInfo
 {
   XftFont *font_info;
 }
--- gnustep-back-0.8.3/Source/xlib/XGContext.m.orig	Tue Jun  4 04:21:35 2002
+++ gnustep-back-0.8.3/Source/xlib/XGContext.m	Sun Dec 29 12:30:06 2002
@@ -78,7 +78,7 @@
 #ifdef HAVE_XFT
   if ([[NSUserDefaults standardUserDefaults] boolForKey: @"GSFontAntiAlias"])
     {
-      fontClass = [XftFontInfo class];
+      fontClass = [GSXftFontInfo class];
     }
 #endif
   if (fontClass == Nil)
--- gnustep-back-0.8.3/Source/xlib/XftFontInfo.m.orig	Tue Oct 22 05:47:15 2002
+++ gnustep-back-0.8.3/Source/xlib/XftFontInfo.m	Sun Dec 29 12:30:21 2002
@@ -44,14 +44,14 @@
  */
 static NSMutableDictionary	*_globalFontDictionary = nil;
 
-@interface XftFontInfo (Private)
+@interface GSXftFontInfo (Private)
 
 - (BOOL) setupAttributes;
 - (XGlyphInfo *)xGlyphInfo: (NSGlyph) glyph;
 
 @end
 
-@implementation XftFontInfo
+@implementation GSXftFontInfo
 
 - initWithFontName: (NSString*)name matrix: (const float *)fmatrix
 {
@@ -305,7 +305,7 @@
 
 @end
 
-@implementation XftFontInfo (Private)
+@implementation GSXftFontInfo (Private)
 
 - (BOOL) setupAttributes
 {
